pywren/wren.py

```diff
@@ -13,6 +13,15 @@
 
 
 def default_executor(**kwargs):
+    """
+    Initialize and return an executor object.
+
+    :param config: Default `None`. This is defaults to `~/.pywren_config`
```

Comment:
defaults to

Reply:
It's a bit confusing whether it defaults to None or `~/.pywren_config`. you can explain that the configs passed here will overwrite `pywren_config`